src/navigation/screenOptions.tsx

```diff
@@ -8,6 +8,8 @@ import NavigationHeaderLeft from '../components/NavigationHeaderLeft'
 import { COLORS } from '../constants/theme/colors'
 import { RoutesParams } from './NavigationTypes'
 
+export const headerHeightPercentage = wp('15%')
```

Comment:
Optional: If `useTabletHeaderHeight` is only ever used with this value it could just be used as default there, couldn't it? Or even converted to a constant instead of a function. Or am I missing something here?

Reply:
Yes I thought about this too, but we cannot use a hook here and therefore I did it like this.

Reply:
See suggestion below on what I meant :) You can use that or just keep the current.